dokka-integration-tests/gradle/src/testExampleProjects/kotlin/ExampleProjectsTest.kt

```diff
@@ -118,6 +122,28 @@ class ExampleProjectsTest {
             ExampleProject.CompositeBuild -> ":build"
             else -> ":dokkaGenerate"
         }
+
+        init {
+            updateGradleProperties()
+        }
+
+        private fun updateGradleProperties() {
+            when (exampleProjectName) {
+                ExampleProject.KotlinMultiplatform -> {
+                    project.gradleProperties {
+                        // kotlin.native.enableKlibsCrossCompilation must be set to `true`
+                        // otherwise Kotlin can't generate a Klib for Coroutines in macosMain
+                        // when generating on Linux machines, resulting in 'Error class: unknown class'
+                        // for CoroutineScope appearing in the generated docs.
+                        kotlin.native.enableKlibsCrossCompilation = true
```

Comment:
is there a reason why we set it here and also in example `gradle.properties?

Reply:
The util that updates the Gradle properties doesn't merge them with the current ones, for better reproducibility

The test-util _used_ to merge the new properties with the old ones, but I took it out because it caused flaky tests https://github.com/Kotlin/dokka/commit/afab82d6755c419547a338e9fa0d552f29c02087#diff-32f73820e3e4e7cd35545577c35d32df857b8936909734336e493f74bd7a696eR37-R40